data/resources/style.css

```diff
@@ -148,6 +148,39 @@ sidebarsearchitemrow {
   color: @accent_fg_color;
 }
 
+messagedocument > label.sender-text {
+  margin-bottom: 3px;
+}
+
+messagedocument label.numeric.dim-label {
```

Comment:
This is really generic, can you add a new css class instead? Something like "label.file-size".